src/core/api/reactiveSearchApi.js

```diff
@@ -24,13 +26,16 @@ function reactiveSearchApi(query, settings) {
     query,
   };
 
+  if (this.mongodb) {
+    Object.assign(body, { mongodb: getMongoRequest(this.app, this.mongodb) });
+  }
 
   return this.performFetchRequest({
     method: 'POST',
     path: '_reactivesearch',
     body,
-    headers: getTelemetryHeaders(this.enableTelemetry),
-    isRSAPI: true,
+    headers: getTelemetryHeaders(this.enableTelemetry, !this.mongodb),
+    isRSAPI: !this.mongodb,
```

Comment:
@mohdashraf010897 How is it related to mongo?

Reply:
https://www.loom.com/share/af739667196a46f3924736a4b311e4df

Reply:
@bietkul 
f5a18e7